src/java.base/share/classes/java/io/RandomAccessFile.java

```diff
@@ -900,17 +900,17 @@ public final int readInt() throws IOException {
      * {@code b1}, {@code b2}, {@code b3},
      * {@code b4}, {@code b5}, {@code b6},
      * {@code b7}, and {@code b8,} where:
-     * <blockquote><pre>
-     *     0 &lt;= b1, b2, b3, b4, b5, b6, b7, b8 &lt;=255,
-     * </pre></blockquote>
+     * {@snippet lang=java :
+     *         0 <= b1, b2, b3, b4, b5, b6, b7, b8 <= 255,
```

Comment:
Same: this is not Java language syntax. Code or pre tags are fine here, they are not deprecated.

Reply:
I would keep the snippet markup but change the language to "text"; removing the expectation of language support.
The benefits of a consistent look are still desirable.

Reply:
Changed to unspecified language in 8ae42cf24b931d21dc643d233f33d4ccc5b32d6f.